src/Core/Util/DateTime/DateTime.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Util\DateTime;
+
+/**
+ * Defines reusable values for DateTime
+ */
+final class DateTime
+{
+    /**
+     * DateTime value which should be considered same as null
+     */
+    const NULL_VALUE = '0000-00-00 00:00:00';
+
+    /**
+     * This class is not supposed to be initialized as its purpose is to define constants
```

Comment:
I'd say:
- either we dont care that this class can be initialized, because it's useless so if a module/agency developer uses it, I dont see what usage he can do about it 😄 
- if we dont want people to be mistaken between `\DateTime` and `PrestaShop\PrestaShop\Core\Util\DateTime\DateTime` then we can even further throw a RuntimeException in the __construct() with the error message